### ISSUE TITLE
Tmog is not added on tmog list (have whitemane+)

### ISSUE THREAD

**Issue by Hirosee** [gamefreedomgit/Frostmourne#2059]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/wotlk)

**Description:**  i have Whitemane+ account name: Mirei, have Vez 25hm 2H Sword for tmog but on tmog npc can't see it on tmog list

**How to reproduce:**
add Vez 25hm 2H Sword on tmog list

**How it should work:**
after i traded weapon it should have in my list

**Database links:**

![Screenshot 2024-10-06 043452](https://github.com/user-attachments/assets/6f80168f-0ae7-4ec0-a4bf-848630d5b394)

**Comment by Afgann** [comment]
@Hirosee did you check all 13 pages? There is only screenshot of one page.

**Comment by Hirosee** [comment]
Yes, items from my other toon in same acc is not presented on Tmog list, i have checked it throughly
